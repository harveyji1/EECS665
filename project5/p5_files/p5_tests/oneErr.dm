a:int;
b:bool;
f : () void {
	a = b;
}
