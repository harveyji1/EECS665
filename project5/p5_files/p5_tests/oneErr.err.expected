FATAL [5,2]-[5,7]: Invalid assignment operation
Type Analysis Failed
